step-templates/ansible-run-playbook.json

```
@@ -0,0 +1,34 @@
+{
+  "Id": "f0af011b-f99d-4809-84fe-af005b619fa2",
+  "Name": "Run Ansible Playbook",
```

Since the script type isn't present in the library can you add the script type to the name - it just helps people to identify which language its written in. I added a suggestion but happy if you dont like parentheses and want to use something different :smile: 
```suggestion
  "Name": "Run Ansible Playbook (bash)",
```